JITImpl/TCC/CMakeLists.txt

```diff
@@ -67,7 +67,8 @@ if (NOT EXISTS ${PROJECT_BINARY_DIR}/tcc_${TCCJIT_ARCH})
         file(RENAME ${PROJECT_BINARY_DIR}/${TCC_PKG} ${PROJECT_BINARY_DIR}/tcc_unix)
         file(REMOVE ${PROJECT_BINARY_DIR}/${TCC_PKG}.tar.bz2)
         file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/tcc_unix/build)
-        execute_process(COMMAND ${PROJECT_BINARY_DIR}/tcc_unix/configure --extra-cflags=-fPIC
+        set(ENV{CFLAGS} "-fPIC -Og -g -Wall")
```

Comment:
env vars are not scoped in cmake, so this might have unwanted effects on other sub processes. you could backup-restore it, or clearing afterwards should be fine

Reply:
fixed with 7dd136a

Reply:
true, fixed with 7dd136a